datm/atm_comp_nuopc.F90

```diff
@@ -292,7 +297,8 @@ subroutine InitializeAdvertise(gcomp, importState, exportState, clock, rc)
        call datm_datamode_core2_advertise(exportState, fldsExport, flds_scalar_name, &
             flds_co2, flds_wiso, flds_presaero, rc)
        if (ChkErr(rc,__LINE__,u_FILE_u)) return
-    case ('CORE2_IAF_JRA')
+    case ('CORE_IAF_JRA')
+       write(6,*)'DEBUG: calling datm_datamode_jra_advertise'
```

Comment:
remove this line or put it in #ifdef DEBUG

Reply:
done